src/probnum/quad/kernel_embeddings/_kernel_embedding.py

```diff
@@ -82,48 +91,48 @@ def kernel_variance(self) -> float:
+    def _get_kernel_embedding(
+        kernel: Kernel, measure: IntegrationMeasure
+    ) -> Tuple[Callable, Callable]:
+        """Select the right kernel embedding given the kernel and integration measure.
 
-def _get_kernel_embedding(
-    kernel: Kernel, measure: IntegrationMeasure
-) -> Tuple[Callable, Callable]:
-    """Select the right kernel embedding given the kernel and integration measure.
-
-    Parameters
-    ----------
-    kernel :
-        Instance of a kernel.
-    measure :
-        Instance of an integration measure.
-
-    Returns
-    -------
-    kernel_mean :
-        The kernel mean function.
-    kernel_variance :
-        The kernel variance function.
+        Parameters
+        ----------
+        kernel :
+            Instance of a kernel.
+        measure :
+            Instance of an integration measure.
 
-    Raises
-    ------
-    NotImplementedError
-        If the given kernel is unknown.
-    NotImplementedError
-        If the kernel embedding of the kernel-measure pair is unknown.
-    """
+        Returns
+        -------
+        kernel_mean :
+            The kernel mean function.
+        kernel_variance :
+            The kernel variance function.
+
+        Raises
+        ------
+        NotImplementedError
+            If the given kernel is unknown.
+        NotImplementedError
+            If the kernel embedding of the kernel-measure pair is unknown.
+        """
 
-    # Exponentiated quadratic kernel
-    if isinstance(kernel, ExpQuad):
-        if isinstance(measure, GaussianMeasure):
-            return _kernel_mean_expquad_gauss, _kernel_variance_expquad_gauss
-        if isinstance(measure, LebesgueMeasure):
-            return _kernel_mean_expquad_lebesgue, _kernel_variance_expquad_lebesgue
-
-    # Matern
-    if isinstance(kernel, (Matern, ProductMatern)):
-        if isinstance(measure, LebesgueMeasure):
-            return _kernel_mean_matern_lebesgue, _kernel_variance_matern_lebesgue
-
-    # other kernels
-    raise NotImplementedError(
-        "The combination of kernel ({0}) and measure ({1}) is not available as kernel "
-        "embedding.".format(type(kernel), type(measure))
-    )
+        # Exponentiated quadratic kernel
+        if isinstance(kernel, ExpQuad):
+            if isinstance(measure, GaussianMeasure):
+                return _kernel_mean_expquad_gauss, _kernel_variance_expquad_gauss
+            if isinstance(measure, LebesgueMeasure):
+                return _kernel_mean_expquad_lebesgue, _kernel_variance_expquad_lebesgue
+
+        # Matern
+        if isinstance(kernel, (Matern, ProductMatern)):
+            if isinstance(measure, LebesgueMeasure):
+                return _kernel_mean_matern_lebesgue, _kernel_variance_matern_lebesgue
+
+        # other kernels
+        raise NotImplementedError(
+            "The combination of kernel ({0}) and measure ({1}) is not available as "
+            "kernel embedding.".format(type(kernel), type(measure))
```

Comment:
```suggestion
            f"The combination of kernel ({type(kernel)}) and measure ({type(measure)}) is not available as "
            f"kernel embedding."
```

f-Strings tend to be slightly more readable in code.